app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -1529,6 +1529,9 @@ protected function _saveProducts()
             $existingImages = $this->getExistingImages($bunch);
 
             foreach ($bunch as $rowNum => $rowData) {
+                // reset category processor's failed categories array
+                $this->categoryProcessor->clearFailedCategories();
```

Comment:
this change clears failed categories' array for every bunch, avoiding to log the same failed categories every time a bunch is imported. See `app/code/Magento/CatalogImportExport/Model/Import/Product.php::processRowCategories` that is called for every bunch's import called in the same file at line 1603